installer/conf/sudoers

```diff
@@ -32,10 +32,17 @@ omsagent ALL=(ALL) NOPASSWD: /opt/microsoft/omsagent/ruby/bin/ruby /opt/microsof
 omsagent ALL=(ALL) NOPASSWD: /opt/microsoft/omsagent/ruby/bin/ruby /opt/microsoft/omsagent/bin/hdinsightmanifestreader.rb
 
 #run update management commands
-omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get update 
-omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get dist-upgrade * 
-omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get upgrade *  
+omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get dist-upgrade *
+omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get update
+omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get upgrade *
+omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get * update
+omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get * dist-upgrade
+omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get * upgrade
+omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get * install
```

Comment:
What's the use of this cmd in the runbook? I thought we explicitly didn't want to whitelist install (I though we agreed on only whitelisting --only-upgrade install...